python/lsst/qserv/tests/benchmark.py

```diff
@@ -145,6 +145,9 @@ def runQueries(self, stopAt=MAX_QUERY):
                                       str(pragmas))
                     column_names = 'noheader' not in pragmas
                     sqlInterface.execute(qText, outFile, column_names)
+                    if 'sortresult' in pragmas:
+                        sortcmd = 'sort %s -o %s' % (outFile, outFile)
+                        os.system(sortcmd)
```

Comment:
Hi Tim,
Planning on replacing those lines with the following -

```
if 'sortresult' in pragmas:
    f = open(outFile,"r")
    sortedLines = sorted(f.readlines())		
    f.close()
    f = open(outFile,"w")
    for line in sortedLines:
        f.write(line)
    f.close()
```
Any suggestions on making it shorter (if possible)? Also, what's the one-line check you had in mind for checking the return value?